src/components/PostEditor/PostEditor.js

```diff
@@ -725,13 +804,162 @@ class PostEditor extends React.Component {
+                        this.setState({
+                          post: { ...post, proposalOptions: newOptions },
+                          valid: this.isValid({ proposalOptions: newOptions })
+                        })
+                      }}
+                    />
+                  </div>
+                ))}
+                <div styleName='proposalOption' onClick={() => this.handleAddOption()}>
+                  <Icon name='Plus' styleName='icon-plus' blue />
+                  <span styleName='optionText'>{t('Add an option to vote on...')}</span>
+                </div>
+                {!isEqual(proposalOptions, this.props.post.proposalOptions?.items) && (
+                  <div styleName='proposalOption warning' onClick={() => this.handleAddOption()}>
+                    <Icon name='Hand' styleName='icon-plus' />
+                    <span styleName='optionText'>{t('If you save changes to options, all votes will be discarded')}</span>
+                  </div>
+                )}
+                {proposalOptions.length === 0 && (
+                  <div styleName='proposalOption warning' onClick={() => this.handleAddOption()}>
+                    <Icon name='Hand' styleName='icon-plus' />
+                    <span styleName='optionText'>{t('Proposals require at least one option')}</span>
+                  </div>
+                )}
+              </div>
+            </div>
+          )}
+          {isProposal && (
+            <div styleName='footerSection'>
+              <div styleName='footerSection-label'>{t('Proposal type')}</div>
+
+              <div styleName='inputContainer'>
+                <Dropdown
+                  styleName='dropdown'
+                  toggleChildren={
+                    <span styleName='dropdown-label'>
+                      {proposalType ? t(`${proposalType}`) : t('Select pre-set')}
+                      <Icon name='ArrowDown' blue />
+                    </span>
+                  }
+                  items={[
+                    { label: t('Single Vote per person'), onClick: () => this.handleSetProposalType(PROPOSAL_TYPE_SINGLE) },
+                    { label: t('Multi-vote, unrestricted'), onClick: () => this.handleSetProposalType(PROPOSAL_TYPE_MULTI_UNRESTRICTED) },
+                    { label: t('Majority'), onClick: () => this.handleSetProposalType(PROPOSAL_TYPE_MAJORITY) }
+                  ]}
+                />
+              </div>
+            </div>
+          )}
+          {isProposal && (
+            <div styleName='footerSection'>
+              <div styleName='footerSection-label'>{t('Quorum')}</div>
+              <SliderInput percentage={post.quorum} setPercentage={this.handleSetQuorum} />
+            </div>
+          )}
+          {isProposal && (
+            <AnonymousVoteToggle
+              isAnonymousVote={!!post.isAnonymousVote}
+              toggleAnonymousVote={this.toggleAnonymousVote}
+            />
+          )}
+          {/* {isProposal && (
```

Comment:
to discuss: can casual or strict be toggled just by whether there is a timeline or not? then we wouldn't need another control. would need to explain well though